packages/frontend-api/src/Model/Mutation/Customer/DeliveryAddress/DeliveryAddressApiDataFactory.php

```diff
@@ -8,17 +8,17 @@
 use Shopsys\FrameworkBundle\Model\Country\CountryFacade;
 use Shopsys\FrameworkBundle\Model\Customer\Customer;
 use Shopsys\FrameworkBundle\Model\Customer\DeliveryAddressData;
-use Shopsys\FrameworkBundle\Model\Customer\DeliveryAddressDataFactory as FrameworkProductRepository;
+use Shopsys\FrameworkBundle\Model\Customer\DeliveryAddressDataFactory as FrameworkDeliveryAddressDataFactory;
```

Comment:
the alias is unnecessary now :wink:

Reply:
I Don't mind it, but it was easy pick, so I changed it